catalog/acm/setters.yaml

```diff
@@ -24,7 +24,7 @@ data:
   # The namespace in which to create Kubernetes CRDs representing cluster resources e.g. config-control
   platform-namespace: config-control
   # The project in which to manage cluster resources
-  platform-project-id: platform-project-id
+  project-id: project-id
   # The URL for the repo for ACM (Config Sync)
   sync-repo: https://source.developers.google.com/p/{project_id}/r/{repo}
   # The reference for the repo e.g. projects/${project}}/repos/${name}. Used to grant access to the ACM SA
```

Comment:
These two setters can also be interpolated for simplicity - we will still need repo_project_id and repo_name

Reply:
yes, we had discussed this earlier. Instead of interpolating it's better to ask explicitly for this. That way for non CSR repos it can be directly supplied. I will remove the curly braces in the example name, so as not to imply interpolation.